apisix/ssl.lua

```diff
@@ -56,6 +63,12 @@ function _M.server_name()
     return sni
 end
 
+function _M.set_protocols_by_clienthello(ssl_protocols)
+    if ssl_protocols then
+       return ngx_ssl_client.set_protocols(ssl_protocols)
+    end
+    return true
+end
```

Comment:
```suggestion

function _M.set_protocols_by_clienthello(ssl_protocols)
    if ssl_protocols then
       return ngx_ssl_client.set_protocols(ssl_protocols)
    end
    return true
end

```
Please follow the [code style](https://github.com/apache/apisix/blob/master/CODE_STYLE.md)